.circleci/config.yml

```diff
@@ -125,14 +130,18 @@ jobs:
           name: Test Packages
           command: go test -v -a -timeout 15m -ldflags '-s' -coverprofile=coverage_pkg.txt -covermode=atomic -coverpkg=./pkg/... ./pkg/...
           no_output_timeout: 15m
+
       - run:
-          name: Test Commands
-          # Commands cannot be tested with -coverpkg due to https://github.com/golang/go/issues/23910, hence they are tested separately.
-          command: go test -v -a -timeout 15m -ldflags '-s' -coverprofile=coverage_cmd.txt -covermode=atomic ./cmds/...
-          no_output_timeout: 15m
+          name: Test coverage
+          command: |
+              UROOT_QEMU_COVERPROFILE=vmcoverage.txt go test -timeout=15m \
+                -coverprofile=coverage.txt -covermode=atomic -cover \
+                ./cmds/... ./pkg/...
```

Comment:
Don't split this onto multiple lines, but leave the "Test Packages" line above intact. It's inconsistent.

Reply:
Did the yaml linter complain about the line length?

Reply:
I just rebased the old commits, will adjust that